iml-api/src/graphql.rs

```diff
@@ -136,6 +137,86 @@ impl Deref for SortDir {
     }
 }
 
+#[derive(GraphQLEnum, sqlx::Type)]
+#[repr(i16)]
+enum MessageClass {
+    Normal = 0,
+    Lustre = 1,
+    LustreError = 2,
+    Copytool = 3,
+    CopytoolError = 4,
+}
+
+impl From<i16> for MessageClass {
+    fn from(value: i16) -> Self {
+        match value {
+            0 => MessageClass::Normal,
+            1 => MessageClass::Lustre,
+            2 => MessageClass::LustreError,
+            3 => MessageClass::Copytool,
+            4 => MessageClass::CopytoolError,
+            _ => panic!("Invalid variant"),
```

Comment:
Would it be ok to issue the panic if at some time we decide to extend the enum?
Is it possible, when the database can has value `5`, and the clients have the versions a bit older?

Reply:
We guarantee correctness at insertion. If the database is migrated it's all good.

Reply:
Let it be, then.